doc/nvim-treesitter.txt

```diff
@@ -1,6 +1,7 @@
 *nvim-treesitter*  Treesitter configurations and abstraction layer for Neovim.
 
-Minimum version of neovim: nightly
+Minimum version of Neovim: latest stable
+Recommended version of Neovim: nightly
```

Comment:
This change looks unrelated to this PR (and unintended; nvim-treesitter is experimental and not meant to target stable).

Reply:
Ah. It looks like the README has been updated since this was written. (1c9a546fcdbec3cb8af0143409aa527d8e5ec7a9 recommended «Neovim latest stable version or [nightly](https://github.com/neovim/neovim#install-from-source)»; a189323454d1215c682c7ad7db3e6739d26339c4 changed that to «**Neovim 0.7.0 or later** (latest [nightly](https://github.com/neovim/neovim#install-from-source) recommended)»; ee2dcc1cbe28fa605bd9a26191b3d6ac90f0fa24 changed that to «**Neovim 0.7.0 or later** built with **tree-sitter 0.20.3+** (latest [nightly](https://github.com/neovim/neovim#install-from-source) recommended)».)

Would you prefer that this is updated to
```
Minimum version of Neovim: 0.7.0 or later
Recommended version of Neovim: nightly
Minimum version of tree-sitter: 0.20.3 or later
```
or that the patch is dropped from this set entirely? I'm not too attached to getting this in here; this was something I noticed while making other changes and figured would be a quick fixup that could be bundled along without trouble.

Reply:
Yes, please leave it the way it is on master.

Reply:
Patch dropped.